internal/cmd/ansible-operator/run/cmd.go

```diff
@@ -99,12 +99,26 @@ func run(cmd *cobra.Command, f *flags.Flags) {
 		}
 	}
 
+	// We need to check both because EnableLeaderElection is deprecated
+	// If one of them is true than it means that the default value was changed
+	// todo:remove it for 2.0.0
+	if f.LeaderElection || f.EnableLeaderElection {
+		f.LeaderElection = true
+	}
```

Comment:
You should error out if both are set instead.
```suggestion
	if cmd.Flags().Changed("leader-elect") && cmd.Flags().Changed("enable-leader-election") {
		return errors.New("only one of --leader-elect and --enable-leader-election may be set")
	}
```